x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -34673,20 +34673,6 @@
     "xpack.serverlessSearch.configureClient.title": "Configurer votre client",
     "xpack.serverlessSearch.disabled": "Désactivé",
     "xpack.serverlessSearch.enabled": "Activé",
-    "xpack.serverlessSearch.footer.billing.title": "Facturation et utilisation",
-    "xpack.serverlessSearch.footer.community.title": "Rejoindre la communauté",
-    "xpack.serverlessSearch.footer.discoverCard.buttonText": "Explorer les données dans Discover",
-    "xpack.serverlessSearch.footer.discoverCard.description": "Avec Discover, vous pouvez rapidement rechercher et filtrer vos données, obtenir des informations sur la structure des champs et afficher vos résultats dans une visualisation.",
-    "xpack.serverlessSearch.footer.discoverCard.title": "Explorer et visualiser vos données dans Discover",
-    "xpack.serverlessSearch.footer.feedback.title": "Donner un retour",
-    "xpack.serverlessSearch.footer.inviteUsers.title": "Inviter d'autres utilisateurs",
-    "xpack.serverlessSearch.footer.pipelinesCard.buttonText": "Configurer vos pipelines d’ingestion",
-    "xpack.serverlessSearch.footer.pipelinesCard.description": "Prétraiter vos données avant leur indexation dans Elasticsearch. Supprimez des champs, extrayez des valeurs de textes ou enrichissez vos données avec des modèles de Machine Learning comme ELSER.",
-    "xpack.serverlessSearch.footer.pipelinesCard.title": "Transformer vos données à l’aide de pipelines",
-    "xpack.serverlessSearch.footer.searchUI.buttonText": "Créer avec Search UI",
-    "xpack.serverlessSearch.footer.searchUI.description": "L’interface utilisateur Search est une bibliothèque JavaScript libre et gratuite maintenue par Elastic pour un développement rapide d’expériences de recherche modernes et attrayantes.",
-    "xpack.serverlessSearch.footer.searchUI.title": "Créer une interface utilisateur avec Search UI",
-    "xpack.serverlessSearch.footer.title": "Et ensuite ?",
```

Comment:
I think we need to add translation for all of the `xpack.serverlessSearch.overview.footer.*` translations we added in order for this to be displayed properly?

Reply:
These are all the old ones removed by running `scripts/i18n_check --fix` since they are not longer in use.

We don't add the new translations that I'm aware of. That gets handled separately. Not sure what kind of schedule we're on for serverless for that.